homeassistant/components/light/x10.py

```diff
@@ -79,7 +72,7 @@ def brightness(self):
     @property
     def is_on(self):
         """Return true if light is on."""
-        return self._is_on
+        return bool(get_unit_status(self._id))
```

Comment:
We should only do I/O in the update method. Not inside the properties. Please add the update method back.